src/utils/sformat.fs

```diff
@@ -916,9 +918,9 @@ namespace Microsoft.FSharp.Text.StructuredFormat
             match c with 
```

Comment:
The relevant part of the spec will be the part on strings - and yes, I believe the ones you've got are the only characters that **must** be escaped.   Even some of those need not be - like ``\t`` in strings - but I think you're aiming for a string that looks "good" rather than full if strange characters.